### ISSUE TITLE
`ng generate route --dry-run` modifies system-config.ts

### ISSUE THREAD

**Issue by antonmoiseev** [angular/angular-cli#753]
1. OS X El Capitan
2. `ng --version`:
  
  ```
  angular-cli: 1.0.0-beta.1
  node: 6.1.0
  os: darwin x64
  ```
3. Run `ng generate route login --dry-run`. After the command completes `system-config.ts` is modified - `'app/+login'`entry is added to the `barrels` array "App specific barrels" section. Expected behavior - in the `--dry-run` mode none of the files should be modified.

**Comment by hannahhoward** [comment]
Actually several files are modified. Git diff after run `ng generate route test --dry-run`:

```
diff --git a/src/app/tour-of-heroes.component.html b/src/app/tour-of-heroes.component.html
index b6931b5..3b8fa5e 100644
--- a/src/app/tour-of-heroes.component.html
+++ b/src/app/tour-of-heroes.component.html
@@ -1,3 +1,5 @@
 <h1>
   {{title}}
 </h1>
+
+<router-outlet></router-outlet>
\ No newline at end of file
diff --git a/src/app/tour-of-heroes.component.ts b/src/app/tour-of-heroes.component.ts
index 88588e2..546f879 100644
--- a/src/app/tour-of-heroes.component.ts
+++ b/src/app/tour-of-heroes.component.ts
@@ -1,11 +1,18 @@
 import { Component } from '@angular/core';
+import { TestComponent } from './+test';
+import { Routes , ROUTER_DIRECTIVES, ROUTER_PROVIDERS} from '@angular/router';
 
 @Component({
   moduleId: module.id,
   selector: 'tour-of-heroes-app',
   templateUrl: 'tour-of-heroes.component.html',
-  styleUrls: ['tour-of-heroes.component.css']
+  styleUrls: ['tour-of-heroes.component.css'],
+  directives: [ROUTER_DIRECTIVES],
+  providers: [ROUTER_PROVIDERS]
 })
+@Routes([
+  {path: '/test', component: TestComponent}
+])
 export class TourOfHeroesAppComponent {
   title = 'tour-of-heroes works!';
 }
diff --git a/src/system-config.ts b/src/system-config.ts
index d2af1aa..bf93acc 100644
--- a/src/system-config.ts
+++ b/src/system-config.ts
@@ -29,6 +29,7 @@ const barrels: string[] = [
   // App specific barrels.
   'app',
   'app/shared',
+  'app/+test',
   /** @cli-barrel */
 ];
 ```

**Comment by Brocco** [comment]
Merged via #762
